frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -106,45 +117,20 @@ export const saveDashboardAndCards = createThunkAction(
         );
       }
 
-      // reposition the cards
-      if (_.some(updatedDashcards, dc => dc.isDirty || dc.isAdded)) {
-        const cards = updatedDashcards.map(
-          ({
-            id,
-            card_id,
-            row,
-            col,
-            size_x,
-            size_y,
-            series,
-            parameter_mappings,
-            visualization_settings,
-          }) => ({
-            id,
-            card_id,
-            row,
-            col,
-            size_x,
-            size_y,
-            series,
-            visualization_settings,
-            parameter_mappings:
-              parameter_mappings &&
-              parameter_mappings.filter(
-                mapping =>
-                  // filter out mappings for deleted parameters
-                  _.findWhere(dashboard.parameters, {
-                    id: mapping.parameter_id,
-                  }) &&
-                  // filter out mappings for deleted series
-                  (!card_id ||
-                    card_id === mapping.card_id ||
-                    _.findWhere(series, { id: mapping.card_id })),
-              ),
-          }),
-        );
-
-        const result = await DashboardApi.reposition_cards({
+      // update the dashboard cards
+      if (_.some(updatedDashcards, dc => dc.isDirty)) {
```

Comment:
We don't need to update dashcards where `isAdded` is true anymore. That data is already stored.